EgammaAnalysis/ElectronTools/interface/EnergyScaleCorrection_class.h

```diff
@@ -1,5 +1,5 @@
-#ifndef EgammaAnalysis_ElectronTools_EnergyScaleCorrection_class_h
-#define EgammaAnalysis_ElectronTools_EnergyScaleCorrection_class_h
+#ifndef EnergyScaleCorrection_class_h
+#define EnergyScaleCorrection_class_h
```

Comment:
@emanueledimarco 
Do we have a reason why we are not following the CMSSW convention (or rule) for this definition name?

Taejeong